examples/restore-with-preserve-nodeport.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kahu.io/v1beta1
```

Comment:
Why this file is outside directly under examples? Supposed to be in a folder based on the name

Reply:
> Why this file is outside directly under examples? Supposed to be in a folder based on the name

This directory contains yamls with different attributes of backup and restore yamls and sample demo folder contains some specific usecase scenarios